starter.js

```diff
@@ -0,0 +1,176 @@
+//Part 0: just do it
+function justDoIt(name){
+
+    //return `${name}, JUST DO IT!`;
+      return name +", JUST DO IT!";
+    
+    };
+    console.log(justDoIt('SHAHAD'));// return SHAHAD, JUST DO IT!
+    
+    //Part 1: Big or Small String
+    
+    function bigOrSmallString(word){
+    
+    if (word.length>10){
+       
+        return `This word is loooooong!`;
+    }else{
+    
+        return `This word is short`;
+    }
+    };
+    
+    console.log(bigOrSmallString('congratulations'));
+    console.log(bigOrSmallString('awesome'));
+    
+    
+    //PART 2: Odd or Even String Length?
+    
+    function oddOrEvenString(stringLength){
+    if (stringLength.length %2 ==0)
+    {//even
+    return `This string length is TOTALLY even!`;
+    }else if (stringLength.length % 2 == 1)
+    {//odd
+      return `This string is length is odd.`;
+    }
+    }
+    console.log(oddOrEvenString('happines'));
+    console.log(oddOrEvenString('happy'));
+    
+    //PART 3: Median
+    function medianOfArray(){
+    
+        {
+          
+            var median = 0, numsLen = numbers.length;
+            numbers.sort();
+         
+            if (
+                numsLen % 2 === 0 // is even
+            ) {
+                
+                median = (numbers[numsLen / 2 - 1] + numbers[numsLen / 2]) / 2;
+            } else { 
+                median = numbers[(numsLen - 1) / 2];
+            }
+         
+            return median;
+        }
+    
+    }
+    var numbers =[1,2,3,4,5,6,7,8]; 
+    console.log(medianOfArray(numbers));
```

Comment:
you are passing a parameter but your function doesn't have any !! your function needs to have a signature like this `  function medianOfArray(number) {}` , your function worked because you define a global varaiable name  numbers  ` var numbers =[1,2,3,4,5,6,7,8]; `